### PR TITLE
use tflite bilinear op to resize input of label_image

### PR DESCRIPTION
**Pull request by freedomtan**
replace previous naive `downsize()` function with a `resize()` using TF Lite RESIZE_BILINEAR operator

### CODE REVIEW

File: tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h [hunk 1]
```diff
@@ -16,30 +16,74 @@ limitations under the License.
...
+      resolver.FindOp(BuiltinOperator_RESIZE_BILINEAR);
+  interpreter->AddNodeWithParameters({0, 1}, {2}, nullptr, 0, nullptr,
+                                     resize_op, nullptr);
+
+  interpreter->AllocateTensors();
+
+  // fill input image
+  // in[] are integers, cannot do memcpy() directly
+  auto input = interpreter->typed_tensor<float>(0);
+  for (int i = 0; i < number_of_pixels; i++) input[i] = in[i];
```
Comment (andrehentz) [inline tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h]: please add curly braces and use multiple lines.

File: tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h [hunk 2]
```diff
@@ -16,30 +16,74 @@ limitations under the License.
...
-        else
-          out_pixel[c] = in_pixel[c];
-      }
-    }
+void resize(T* out, uint8_t* in, int image_height, int image_width,
+            int image_channels, int wanted_height, int wanted_width,
+            int wanted_channels, Settings* s) {
+
+  int number_of_pixels = image_height * image_width * image_channels;
+  std::unique_ptr<Interpreter> interpreter(new Interpreter);
```
Comment (andrehentz) [inline tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h]: This is a clever idea. I wish we had a more general way to manipulate the original graph and add the extra node directly. That would avoid the need to instantiate a separate interpreter just to resize the image. Anyway, no action required. This is just food for thought.

**Review by andrehentz — APPROVED** [review]

**Comment by rmlarsen** [comment]
@freedomtan Please address @andrehentz 's comments.

File: tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h (hunk 2 above)
Comment (freedomtan) [reply tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h]: This idea is actually stolen from [ReadTensorFromImageFile()](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/main.cc#L114) of the original `tensorflow/examples/label_image/` :-)

File: tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h (hunk 1 above)
Comment (freedomtan) [reply tensorflow/contrib/lite/examples/label_image/bitmap_helpers_impl.h]: OK, will do. Clang-format did it :-)

**Pull request merged**